logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -17,6 +17,23 @@
 import math
 import json
 
+try:
+    import django
+except ImportError:
+    _USE_DJANGO = False
+else:
+    _USE_DJANGO = True
+
+try:
+    import flask
+except ImportError:
+    _USE_FLASK = False
+    flask = None
+else:
+    _USE_FLASK = True
+
```

Comment:
Yeah I added a function to detect the web framework, and print out warning message if multiple or no web framework are found in the environment.